setup.py

```diff
@@ -16,7 +16,7 @@
 
     packages=find_packages(exclude=('tests',)),
 
-    data_files=[('templates', glob('templates/*.j2'))],
+    data_files=[('templates', glob('reconcile/templates/*.j2'))],
```

Comment:
Maybe?

I wonder why they aren't copied in the first place...

```suggestion
    data_files=[('reconcile/templates', glob('reconcile/templates/*.j2'))],
```